content/_data/changelogs/lts.yml

```diff
@@ -7410,6 +7410,28 @@
     message: |-
       Remove the Java Native Runtime (JNR) library from Jenkins core.
 
+  - type: rfe
+    category: developer
+    authors:
+      - basil
+    pr_title: Remove glibc from alpine_jdk11
+    references:
+      - url: https://github.com/jenkinsci/docker/pull/1361
+        title: Docker pull request 1361
+    message: |-
+      Remove unused <code>glibc</code> package from Alpine-based Docker images.
+
+  - type: rfe
+    category: developer
+    authors:
+      - basil
+    pr_title: Avoid additional packages by specifying --no-install-recommends
+    references:
+      - url: https://github.com/jenkinsci/docker/pull/1375
+        title: Docker pull request 1375
+    message: |-
+      Remove unnecessary packages from Debian-based Docker images.
```

Comment:
The two most significant packaging changes to occur recently. They are in the "Compatibility risks" section because they have a nonzero risk of regression. I judged the first important enough to include an upgrade guide entry. I judged the second to be less risky and not worth an upgrade guide entry.